graphql_api/types/repository/repository.py

```diff
@@ -593,3 +612,24 @@ async def resolve_test_results(
         else OrderingDirection.DESC,
         **kwargs,
     )
+
+
+@repository_bindable.field("testResultsHeaders")
```

Comment:
thought we had agreed to keep the name generic vs. "headers" in case design changes or we want to use aspects of this object elsewhere